<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>youke</class>
 <widget class="QDialog" name="youke">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>药品信息查询-浏览</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 26pt &quot;方正舒体&quot;;</string>
       </property>
       <property name="text">
        <string>浏览、查询药品信息</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,6">
         <item>
          <widget class="QComboBox" name="comboBox">
           <item>
            <property name="text">
             <string>浏览</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按编号查询</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按名称查询</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按服用方法查询</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>按功效查询</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
